homeassistant/components/mqtt/client.py

```diff
@@ -918,13 +918,16 @@ def _async_mqtt_on_connect(
             result_code,
         )
 
-        self.hass.async_create_task(self._async_resubscribe())
+        birth = self.conf.get(CONF_BIRTH_MESSAGE, DEFAULT_BIRTH)
```

Comment:
Can we add type hints here?

Reply:
43e98fa15e3ff23c5b58bb4e42340b6fe0af1e14

Reply:
Thnx!